orbit/pkg/packaging/linux_shared.go

```diff
@@ -129,14 +138,6 @@ func buildNFPM(opt Options, pkger nfpm.Packager) (string, error) {
 		log.Debug().Interface("file", c).Msg("added file")
 	}
 
-	// Add empty folders to be created.
-	for _, emptyFolder := range []string{"/var/log/osquery", "/var/log/orbit"} {
-		contents = append(contents, (&files.Content{
-			Destination: emptyFolder,
-			Type:        "dir",
-		}).WithFileInfoDefaults())
-	}
```

Comment:
This was the fix for the https://github.com/fleetdm/fleet/pull/5147.

> I'm basically mimicking nfpm's following code:
https://github.com/goreleaser/nfpm/blob/8937b55499afad001db77aaf5269637d2b281898/nfpm.go#L392-L408

The change looks good, just one thing to check, is the `WithFileInfoDefaults()` call required?

Reply:
Ah, this was a recent fix you made. Think I missed it when I pulled rebased on the latest main. Thanks for the catch.